src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressionPatchFilterElement.java

```diff
@@ -126,25 +125,6 @@ private boolean isLineMatching(AuditEvent event) {
         return result;
     }
 
-    /**
```

Comment:
please add `#` in commit message
`Issue 295: Context strategy Checker patch filter shouldn't contain hardcoded checks`
-->
`Issue #295: Context strategy Checker patch filter shouldn't contain hardcoded checks`

Reply:
done